src/test/components/searchInput.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { BrowserRouter } from 'react-router-dom'
```

Comment:
you can also use [MemoryRouter](https://reacttraining.com/react-router/core/api/MemoryRouter) for testing purposes